File: src/main/java/com/stacked/sigaa_ifc/Questionario.java
package com.stacked.sigaa_ifc;

import java.util.Date;

public class Questionario {
    private long id;
    private String titulo;
    private boolean enviado;
    private Date dataInicio, dataFim;
    private Disciplina disciplina;

    public Questionario(long id, String titulo, boolean enviado, Date dataInicio, Date dataFim, Disciplina disciplina) {
        this.id = id;
        this.titulo = titulo;
        this.enviado = enviado;
        this.dataInicio = dataInicio;
        this.dataFim = dataFim;
        this.disciplina = disciplina;
    }

    public long getId() {
        return id;
    }

    public String getTitulo() {
        return titulo;
    }

    public boolean isEnviado() {
        return enviado;
    }

    public void setEnviado(boolean enviado) {
        this.enviado = enviado;
    }

    public Date getDataInicio() {
        return dataInicio;
    }

    public Date getDataFim() {
        return dataFim;
    }

    public Disciplina getDisciplina() {
        return disciplina;
    }
}

File: src/main/java/com/imawa/sigaaforkotlin/models/Questionario.kt
package com.imawa.sigaaforkotlin.models

import java.util.*

/**
 * Representa um questionário cadastrado em uma disciplina.
 * Esta classe não possui as questões, somente as informações acessíveis sem abrir o questionário.
 */
data class Questionario(
    val id: Long,
    val titulo: String,
    val dataInicio: Date,
    val dataFim: Date,
    val isEnviado: Boolean,
    val disciplina: Disciplina
)
